src/web/views/children-dob.njk

```diff
@@ -25,7 +26,8 @@
         legend: aboutYourChild,
         dateLegend: dateOfBirth,
         removeButton: removeChild
-      }
+      },
+      autofocus: i == children.inputCount - 1 and children.inputCount > 1
```

Comment:
This should use strict equality operator `===`

Reply:
Updated to ===